### PR TITLE
Add ability to save blobs to SHSH Host and TSS Saver

### PR DESCRIPTION
**Pull request by CFSerena**
This pull requests makes it possible to save blobs to TSS Saver and SHSH Host with the device information in Blobsaver
TLDR of how it works: a HTTP POST Request is made to https://tsssaver.1conan.com/v2/api/save.php for TSS Saver and https://api.arx8x.net/shsh3/ for SHSH Host, with the parameters being `ecid, deviceIdentifier, and boardConfig`, If the generator and APNonce information is present in the Blobsaver app, then those are also included in the request parameters.

How the Main Menu looks now (Added a toggle for saving to TSS Saver and SHSH Host, on by default): 
image

If the "Save to TSS Saver" toggle is turned on and the program successfully saves blobs, then the "Successfully saved blobs" Pop up now looks like this 

image

### CODE REVIEW

**Comment by Serena-io** [comment]
@1Conan Please let me know if you have any objections to this
